WcaOnRails/app/views/layouts/_navigation.html.erb

```diff
@@ -27,6 +27,7 @@
             <li><a href="<%= faq_path %>"><%= icon('question-circle') %> <%= t '.faq' %></a></li>
             <li><a href="<%= contact_path %>"><%= icon('envelope') %> <%= t '.contact' %></a></li>
             <li><a href="<%= privacy_path %>"><%= icon('user-secret') %> Privacy</a></li>
+            <li><a href="<%= disclaimer_path %>"><%= icon('exclamation-circle') %>Disclaimer</a></li>
```

Comment:
You're missing a space between the icon and the text, which causes some visual misalignment :s

Reply:
Argh, sorry.